commit 59779be03ee38e47ca48808f2d08997a81c3f095
Author: shiju.jose@huawei.com <shiju.jose@huawei.com>
Date:   Tue Jan 16 15:45:34 2018 +0000

    arm64: defconfig: enable CONFIG_ACPI_APEI_EINJ
    
    Enable APEI EINJ for ARM64 to support the error injection.
    
    Signed-off-by: Shiju Jose <shiju.jose@huawei.com>
    Acked-by: Timur Tabi <timur@codeaurora.org>
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 39eb62e19cd7..8d122fc70ac3 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -573,6 +573,7 @@ CONFIG_ACPI_APEI=y
 CONFIG_ACPI_APEI_GHES=y
 CONFIG_ACPI_APEI_PCIEAER=y
 CONFIG_ACPI_APEI_MEMORY_FAILURE=y
+CONFIG_ACPI_APEI_EINJ=y
 CONFIG_EXT2_FS=y
 CONFIG_EXT3_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y